oscore-cf/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.californium</groupId>
+    <artifactId>parent</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>oscore-cf</artifactId>
+  <dependencies>
+	<dependency>
+		<groupId>com.augustcellars.cose</groupId>
+		<artifactId>cose-java</artifactId>
+		<version>0.9.7</version>
+	</dependency>
+	<dependency>
+		<groupId>org.eclipse.californium</groupId>
```

Comment:
dependencies to californium modules are usually done with this pattern:

```
	<dependency>
		<groupId>${project.groupId}</groupId>
		<artifactId>scandium</artifactId>
	</dependency>
	<dependency>
		<groupId>${project.groupId}</groupId>
		<artifactId>californium-core</artifactId>
	</dependency>
```

All versions used are intended to be defined as properties in the parent POM.

```
	<properties>
                ...
		<junit.version>4.12</junit.version>
		<logback.version>1.2.3</logback.version>
		<mockito.version>1.9.5</mockito.version>
		<slf4j.version>1.7.25</slf4j.version>
	</properties>

```

Reply:
I see. This will be changed.

Reply:
I am unsure what you mean with the second thing. If you look in the Effective pom for the oscore pom file you will see that the properties field exists with tis content:

<properties>
    <project.build.javaVersion>1.7</project.build.javaVersion>
    <mockito.version>1.9.5</mockito.version>
    <logback.version>1.2.3</logback.version>
    <hamcrest.version>1.3</hamcrest.version>
    <slf4j.version>1.7.25</slf4j.version>
    <snapshotDependencyAllowed>true</snapshotDependencyAllowed>
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    <junit.version>4.12</junit.version>
  </properties>

Do you mean that I should add the versions of cose, cbor and numbers to this field?

Reply:
```
  	<dependency>
	 	<groupId>com.upokecenter</groupId>
  		<artifactId>cbor</artifactId>
  		<version>2.4.1</version>
  	</dependency>
```

changed to:

```
<properties>
		<cbor.version>2.4.1</cbor.version>
</properties>

<dependency>
	 	<groupId>com.upokecenter</groupId>
  		<artifactId>cbor</artifactId>
  		<version>${cbor.version}</version>
 </dependency>
```

If cbor gets used by more modules, we change it to a managed dependency in the parent POM. That's, how I understand it.

Reply:
My comment failed to paste correctly, but you sorted it out. I understand how you want it now.